app/assets/javascripts/to_do_items.coffee

```diff
@@ -90,24 +100,43 @@ class @TodoItem
     currentCheckbox = $("##{id}")
     currentCheckbox.prop 'checked', !currentCheckbox.prop 'checked'
 
-  showCheckAlert = (listId) ->
-    alertBox = detectList(listId).find('div.alert')
+  showAlert = (listId, typeError) ->
+    alertBox = detectList(listId).find("div.#{typeError}")
     return unless alertBox.css('display') == 'none'
     alertBox.fadeIn('slow').delay(3000).fadeOut('slow')
 
-  renderCheckError = (item) ->
+  onCheckError = (item) ->
     revertChecking item.attr('id')
-    showCheckAlert item.attr('value')
+    showAlert item.attr('value'), 'mark-error'
+
+  onMoveError = (item) ->
+    listId = item.find("input[type='checkbox']").attr('value')
+    revertMoving listId
+    showAlert listId, 'move-error'
 
   doCheckRequest = (item) ->
     $.ajax
       url: item.attr('url')
       method: 'PUT'
       error: () ->
-        renderCheckError item
+        onCheckError item
       success: (response) ->
         toggle response
 
+  changePositionRequest = (item) ->
+    id = extractId item.attr('id')
```

Comment:
Also, I see that previously "do{noun}Request" pattern was used, so you can follow this approach.